# Copyright 2023, The TensorFlow Federated Authors.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

diff --git tensorflow/BUILD tensorflow/BUILD
index b7199470078..1ccd3788f3d 100644
--- tensorflow/BUILD
+++ tensorflow/BUILD
@@ -1005,8 +1005,14 @@
 # Instead, please use public APIs or public build rules TF provides.
 # If you need functionality that is not exposed, we will work with you to expand our public APIs.
 # TODO(b/173549186): Move Google-internal TF code out of learning/brain
+
 package_group(
     name = "internal",
+    packages = ["//..."],
+)
+
+package_group(
+    name = "unused_internal",
     packages = [
         "//devtools/python/indexer/...",
         "//learning/brain/keras/...",